src/main/java/cms/gov/madie/measure/services/ExportService.java

```diff
@@ -0,0 +1,73 @@
+package cms.gov.madie.measure.services;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import cms.gov.madie.measure.utils.ExportFileNamesUtil;
+import org.springframework.stereotype.Service;
+
+import com.nimbusds.oauth2.sdk.util.StringUtils;
```

Comment:
Apache commons is preferred over nimbus DS just for consistency and develop expectations.